_episodes/04-use-license.md

```diff
@@ -5,7 +5,185 @@ exercises: 0
+
+> ### Note: Dual License
+>
+> Some software that have comercial potential adopt a dual license model. They have a version of the code under the GNU General Public License that is freely available to everyone, but they also offer a proprietary version of the code in exchange of some cash. If you decide to adopt this business model, you must be very careful to correctly collect copyright transfer from all contributors to you.
+{: .callout}
+
+In principle, you can create your own bespoke licence, it's your right. However, we strongly discourage it, unless you have appropriate legal background. You are helping the users to understand the licence over your code or data by adopting an existing, widely described licence which better suits with your creation. So, users do not have to analyze the legal details of the licence to understand it.
+
+* Example licences explained in terms of the concepts discussed previously
+https://choosealicense.com/
+
+## Add a LICENCE file to a repository
+
+By including an open source license in your repository, it makes easier for other people to contribute. Most of the projects places their license information in a file named `LICENSE` (`LICENSE.txt` or `LICENSE.md` or `LICENSE.rst` or `COPYING`, `COPYING.txt`, `COPYING.md`, `COPYING.rst` are also used) in the root of the repository.
+
+> ## Challenge 
+> 
+> Add the previously selected license to your GitHub repository
+> 
+> > ## Solution
+> > 
+> > Via the GitHub interface for an already existing repository
+> > 1. On GitHub, navigate to the main page of the repository.
+> > 2. Click on `Create new file` button above the file list
+> > 3. Type `LICENSE` or `LICENSE.md` (with all caps) in the file name field
+> > 4. Click on `Choose a license template` on the right
+> > 5. Click on the selected license
+> > 6. Check that the license terms correspond to the one
+> > 7. Click on `Review and submit` button
+> > 8. Commit the file
+> > 
+> > Locally: 
+> > 1. Navigate to the root of the repository
+> > 2. Open your favorite text editor with a new file
+> > 3. Go to https://choosealicense.com
+> > 4. Select your license by opening it page
+> > 5. Click on `Copy license text to clipboard`
+> > 6. Paste the content in the text editor
+> > 8. Save the file as `LICENSE.md`
+> > 9. Commit the added file
+> > 
+> > 
+> > Via the GitHub interface for a new repository
+> > 1. Start creation of a new repository
+> > 2. Click on `Add a license`
+> > 3. Select the chosen license
+> {:.solution}
+{:.challenge}
+
```

Comment:
It might be worth adding that different communities have different practices. When preparing an R package to be submitted to CRAN the format of the LICENSE file needs to follow a strict format, and be named LICENSE (not LICENSE.md for instance). The usethis package makes this easy by having functions that do the right thing depending on the license you choose: `use_mit_license()`, `use_gpl3_license()`, etc...